store.c

```diff
@@ -3743,11 +3744,12 @@ int *pitchbend;
     bend = 8192; /* corresponds to zero bend */
     }
 if (!microtone) *pitchbend = bend; /* don't override microtone */
-if (comma53) 
 #ifdef MAKAM
  if (comma53) fprintf(fc53,"%c%d ",note,octave+4);
 #endif
  if (comma53) convert_to_comma53 (acc,  &pitch, pitchbend); 
+ /* FIXME */
+ /* indentation implied next 3 lines should be controlled by if (comma53) */
```

Comment:
It was not at all clear what the intent was.  This preserves the actual semantics.  But maybe these fields should be cleared only if comma53.